dwave_networkx/drawing/qubit_layout.py

```diff
@@ -240,6 +259,16 @@ def draw_embedding(G, layout, emb, embedded_graph=None, interaction_edges=None,
         the same vertices in G), and the drawing will display these overlaps as
         concentric circles.
 
+    line_plot : boolean (optional, default False)
+        If line_plot is True, then qubits are drawn as line segments, and edges
+        are drawn either as line segments between qubits, or as circles where
+        two qubits overlap.  In this drawing style, the interpretation the width
+        and node_size parameters (provided in kwargs) determines the area of the
+        circles, and line widths, respectively.  Qubit line segments are given
+        twice the width of edges.  Layout should be a dict of the form
+        {node: ((x0, y0), (y0, x0)), ...} -- instead of coordinates, the nodes
+        are associated with endpoints of n-dimensional line segments.
+
```

Comment:
```suggestion

    kwargs : optional keywords
       See :func:`networkx.draw_networkx` for a description of optional keywords,
       with the exception of the ``pos`` parameter which is not used by this
       function. If ``linear_biases`` or ``quadratic_biases`` are provided,
       any provided ``node_color`` or ``edge_color`` arguments are ignored.

    Examples
    --------
    
    This example illustrates how functions such as :func:`.draw_pegasus` can make use of this internal function.
    
    .. code-block:: python
    
        import dwave_networkx as dnx
        import matplotlib.pyplot as plt
        from dwave_networkx.drawing.qubit_layout import draw_embedding
        G = dnx.chimera_graph(1)
        layout = dnx.chimera_layout(G)
        draw_embedding(G, layout, {0:[0, 4, 5], 1: [1, 6], 2: [2, 6], 3: [3, 7]})

    """
```

This one will need the deletion of lines 272-277 after committing.